package.json

```diff
@@ -12,13 +12,14 @@
     "build:prod:report": "cross-env NODE_ENV=production BUILD_REPORT=true webpack --display-error-details -p --config webpack.config.babel.js",
     "build:prod": "cross-env NODE_ENV=production webpack --display-error-details -p --config webpack.config.babel.js",
     "clean": "rm -rf dist/",
-    "start": "node server/server.js",
-    "watch": "yarn build:dev-server & yarn watch:server",
+    "start": "ts-node server/server.ts",
+    "watch": "yarn build:dev-server & yarn watch:ts",
+    "watch:ts": "ts-node server/server.ts",
     "watch-debug": "yarn build:dev-server & yarn watch:server-debug",
-    "watch:server": "NODE_ENV=development nodemon -e js --watch server server/server.js",
+    "watch:server": "ts-node server/server.ts",
```

Comment:
isn't it the same than start? no watch option here?

Reply:
There is: ts-node-dev
but on macos it doesnt work flawless yet, so i didnt implement it